brut.apktool/apktool-lib/src/main/java/brut/androlib/Androlib.java

```diff
@@ -120,33 +116,34 @@ public void decodeManifestRaw(ExtFile apkFile, File outDir)
         }
     }
 
-    public void decodeManifestFull(ExtFile apkFile, File outDir, ResTable resTable)
-            throws AndrolibException {
+    public void decodeManifestFull(ExtFile apkFile, File outDir, ResTable resTable) throws AndrolibException {
         mAndRes.decodeManifest(resTable, apkFile, outDir);
     }
 
-    public void decodeResourcesRaw(ExtFile apkFile, File outDir)
-            throws AndrolibException {
+    public void decodeResourcesRaw(ExtFile apkFile, File outDir) throws AndrolibException {
         try {
             LOGGER.info("Copying raw resources...");
-            apkFile.getDirectory().copyToDir(outDir, APK_RESOURCES_FILENAMES);
+            Directory directory = apkFile.getDirectory();
+            List<String> apkResourcesFileNames = new ArrayList<>(Arrays.asList(APK_RESOURCES_FILENAMES));
```

Comment:
Couldn't we just load all the AndResGuard res directories into `APK_RESOURCES_FILENAMES` and ensure `copyToDir` handles null/missing directories?